src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -406,6 +406,21 @@ bool Terminal::IsTrackingMouseInput() const noexcept
     return _terminalInput->IsTrackingMouseInput();
 }
 
+// Method Description:
+// - If the clicked text is a hyperlink, open it
+// Arguments:
+// - The position of the clicked text
+std::wstring Terminal::GetHyperlink(const COORD position)
+{
+    auto attr = _buffer->GetCellDataAt(_ConvertToBufferCell(position))->TextAttr();
+    if (attr.IsHyperlink())
+    {
+        auto uri = _buffer->GetHyperlinkUriFromId(attr.GetHyperlinkId());
+        return uri;
+    }
+    return L"";

```

Comment:
```suggestion
    return {};
```

Doing it this way will just invoke the default constructor for `std::wstring` instead of having a `wchar_t*` somewhere that's just a `\0` getting passed into the constructor.

They'll have the same effect, but theoretically the default constructor should be a teensy bit faster and more memory (and disk space) efficient. (Presuming the optimizer didn't figure all of that out and make them equivalent anyway.)